main.css

```diff
@@ -0,0 +1,219 @@
+body {
```

Comment:
# [REQUIRED] - GENERAL FEEL
The general feel of your page is not looking exactly the same as in the original page. Have you noticed that?
-
They are slightly different, but the good news is that we can change that.

This is what I am currently seeing.
![intuit_clone](https://user-images.githubusercontent.com/22345961/78132541-97589b80-7425-11ea-9e29-85bb5bd0fa02.png)

I would love to see something close to this:

![intuit_original](https://user-images.githubusercontent.com/22345961/78132649-ca029400-7425-11ea-841a-4a7bfc86cf77.png)

What is required is general styling.

Reply:
I changed the width to fixed px from percentage. It should look similar now